projects/1.18/assets/beyond-earth/beyond_earth/lang/zh_cn.json

```
@@ -0,0 +1,342 @@
+{
+  "item.beyond_earth.fuel_bucket": "燃油桶",
+  "item.beyond_earth.oil_bucket": "石油桶",
+  "item.beyond_earth.star_crawler_spawn_egg": "星际爬行者刷怪蛋",
+  "item.beyond_earth.steel_engine": "钢引擎",
+  "item.beyond_earth.oxygen_tank": "氧气罐",
+  "item.beyond_earth.iron_plate": "铁板",
+  "item.beyond_earth.desh_plate": "戴斯板",
+  "item.beyond_earth.space_station": "空间站",
+  "item.beyond_earth.compressed_ostrum": "致密紫金",
+  "item.beyond_earth.engine_frame": "引擎框架",
+  "item.beyond_earth.ostrum_tank": "紫金燃油罐",
+  "item.beyond_earth.desh_engine": "戴斯引擎",
+  "item.beyond_earth.hammer": "锤子",
+  "item.beyond_earth.ostrum_engine": "紫金引擎",
+  "item.beyond_earth.alien_zombie_spawn_egg": "外星僵尸刷怪蛋",
+  "item.beyond_earth.ostrum_ingot": "紫金锭",
+  "item.beyond_earth.desh_tank": "戴斯燃油罐",
+  "item.beyond_earth.rocket_nose_cone": "火箭鼻锥",
+  "item.beyond_earth.compressed_steel": "致密钢板",
+  "item.beyond_earth.steel_ingot": "钢锭",
+  "item.beyond_earth.wheel": "轮子",
+  "item.beyond_earth.steel_tank": "钢罐",
+  "item.beyond_earth.pygro_spawn_egg": "僵尸猪灵刷怪蛋",
+  "item.beyond_earth.pygro_brute_spawn_egg": "外星猪灵蛮兵刷怪蛋",
+  "item.beyond_earth.iron_stick": "铁棍",
+  "item.beyond_earth.oxygen_gear": "氧气套装",
```

```suggestion
  "item.beyond_earth.oxygen_gear": "氧气计",
```
稍微想了想，原文是`Oxygen Gear`，改成了这个（1.16库内已有翻译为`氧气计`）

---

参照星系4翻译为供氧设备了